File: 프로그래머스/0/120921. 문자열 밀기/문자열 밀기.java
class Solution {
    public int solution(String A, String B) {
        int answer = -1;
        int len = A.length();

        for (int i = 0; i < len; i++) {
            if (A.equals(B)) {
                answer = i;
                break;
            }

            char lastChar = A.charAt(len - 1);
            A = lastChar + A.substring(0, len - 1);
        }

        return answer;
    }
}

File: 프로그래머스/0/120921. 문자열 밀기/문자열 밀기.py
def solution(A, B):
    answer = -1
    cnt = 0
    while(cnt <=len(A)):
        if(A == B):
            answer = cnt
            break
        else:
            cnt+=1
            A = A[len(A)-1]+A[:len(A)-1]
            print(A)
    return answer